### PR TITLE
Starting doc overhaul

### PR DESCRIPTION
**Pull request by qiqiliang**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -220,9 +223,9 @@
 
 sphinx_gallery_conf = {
     # path to your examples scripts
-    'examples_dirs': '../../examples',
+    'examples_dirs': ['../../examples/howto_examples', '../../examples/tutorial_examples'],
     # path where to save gallery generated examples
-    'gallery_dirs': 'auto_examples',
+    'gallery_dirs': ['auto_examples', 'howto_examples', 'tutorial_examples'],
```
Comment (arokem) [inline docs/source/conf.py:228]:
```suggestion
    'gallery_dirs': ['howto_examples', 'tutorial_examples'],
```

File: docs/source/reference/tracking.rst [hunk 2]
```diff
@@ -1,3 +1,5 @@
+Move to reference
```
Comment (arokem) [inline docs/source/reference/tracking.rst:1]: I think we can delete this now :-) 
```suggestion
```

File: examples/tutorial_examples/cloudknot_example.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+One way to gain access to massive computational power is by using
+cloud computing. Here, we will demonstrate
+how to use ``pyAFQ`` in the Amazon Web Services cloud.
+
+We will rely on the `AWS Batch Service <https://aws.amazon.com/batch/>`_ ,
+and we will submit work into AWS Batch using software that our group
+developed called `Cloudknot <https://nrdg.github.io/cloudknot/>`_.
+"""
+
+# ##########################################################################
```
Comment (arokem) [inline examples/tutorial_examples/cloudknot_example.py:19]: This can all be uncommented.

File: examples/tutorial_examples/plot_afq_reco80.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+"""
+==========================
+RecoBundles80 using AFQ API
+==========================
+
+An example using the AFQ API to run recobundles with the
+`80 bundle atlas <https://figshare.com/articles/Advanced_Atlas_of_80_Bundles_in_MNI_space/7375883>`_.
+
+"""
+# import os.path as op
```
Comment (arokem) [inline examples/tutorial_examples/plot_afq_reco80.py:10]: This can all be uncommented

File: examples/tutorial_examples/plot_baby_afq.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+.. note::
+
+  This example and resulting pyAFQ support for pediatric bundles was
+  inspired by and largely due to the work of Grotheer et al. [Grotheer2021]_.
+
+  This example uses the Yeatman et al. waypoint ROI approach, first
+  described in [Yeatman2012]_ and further elaborated in [Yeatman2014]_.
+
+"""
+# import os.path as op
```
Comment (arokem) [inline examples/tutorial_examples/plot_baby_afq.py:17]: This can all be uncommented

File: examples/tutorial_examples/plot_bids_layout.py [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+    the draft implementation of the BIDS DWI derivatives available
+    `here <https://bids-specification.readthedocs.io/en/wip-derivatives/05-derivatives/05-diffusion-derivatives.html>`_
+
+In this example, we will explore the use of BIDS in pyAFQ and see
+how BIDS allows us to extend and provide flexibility to the users
+of the software.
+
+"""
+
+# import os
```
Comment (arokem) [inline examples/tutorial_examples/plot_bids_layout.py:25]: Same here

File: examples/tutorial_examples/plot_optic_radiations.py [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+cortex. Studying the optic radiations with dMRI provides a linkage between white
+matter tissue properties, visual perception and behavior, and physiological
+responses of the visual cortex to visual stimulation.
+
+We start by importing some of the components that we need for this example and
+fixing the random seed for reproducibility
+
+"""
+
+# import os.path as op
```
Comment (arokem) [inline examples/tutorial_examples/plot_optic_radiations.py:22]: Same here. Etc.

File: examples/tutorial_examples/plot_tract_profile.py [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+in [Yeatman2014]_.
+
+This example goes into the nitty gritty of the underlying methods. This example
+is provided as a way to understand what goes on under the hood, but for most
+practical uses of pyAFQ, we recommend against using the methods directly,
+instead directing users to use the `GroupAFQ` API, demonstrated in other
+examples.
+
+"""
+# import os.path as op
```
Comment (arokem) [inline examples/tutorial_examples/plot_tract_profile.py:18]: OK - last one.

**Comment by 36000** [comment]
@arokem I think this is ready to merge?

**Comment by arokem** [comment]
OK - let's merge and then go through and clean up / fill in as needed.

**Comment by arokem** [comment]
Congratulations @qiqiliang on all your great work on this pull request! 🎉

**Comment by qiqiliang** [comment]
Awesome! Thank you!!

**Pull request merged**